test/test_rb.py

```diff
@@ -157,3 +158,32 @@ def test_interleaved_rb_experiment(self, interleaved_element: "Gate", qubits: li
         self.validate_metadata(exp_circuits, exp_attributes)
         self.validate_circuit_data(exp_data, exp_attributes)
         self.is_identity(exp_circuits)
+
+
+# @ddt
+# class TestRBUtilities(QiskitTestCase):
+#     """
+#     A test class for additional functionality provided by the RBExperiment
+#     class.
+#     """
+#
+#     def test_epg_computation(self):
```

Comment:
Why the test is disabled?

Reply:
Since it was statistically failing on some testing enviroments. It should be handled in another PR.

Reply:
It behaves differently on different Python versions; until it is resolved I don't want to add it.

Reply:
I think you can add test for each `RBUtil` method with fake input data. No integration test is necessary for testing utils. With latest commits, these methods require no backend to run, thus you can easily prepare the test data.